frontend/lib/src/components/transaction_component/transaction_component.dart

```diff
@@ -0,0 +1,159 @@
+                ? 'Please make sure to input valid Email and valid amount'
+                : 'Please make sure to input valid amount',
+          ),
+          actions: [
+            TextButton(
+              onPressed: () {
+                Navigator.pop(ctx);
+              },
+              child: const Text('Okay'),
+            ),
+          ],
+        ),
+      );
+      return;
+    }
+
+    Navigator.pop(context);
+  }
+
+  @override
+  void dispose() {
+    amountController.dispose();
+    emailController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final formKey = GlobalKey<FormBuilderState>();
+
+    return SingleChildScrollView(
+      padding: const EdgeInsets.fromLTRB(25, 30, 25, 0),
+      child: FormBuilder(
+        key: formKey,
+        autovalidateMode: AutovalidateMode.always,
+        onChanged: () {
+          formKey.currentState!.save();
+        },
+        child: Column(
+          children: [
+            Text(
+              widget.label,
+              style: Theme.of(context).textTheme.labelLarge!.copyWith(
+                    fontWeight: FontWeight.bold,
+                  ),
+            ),
+            const SizedBox(
+              height: 13,
+            ),
+            if (widget.type == 'TRANSFER') ...[
+              InputField(
+                name: 'email',
+                label: 'Email',
+                validator: FormBuilderValidators.email(),
+                controller: emailController,
+                inputType: TextInputType.emailAddress,
+              ),
+              const SizedBox(
+                height: 32,
+              ),
+            ],
+            InputField(
+              name: 'amount',
+              label: 'Amount',
+              validator: FormBuilderValidators.integer(),
+              controller: amountController,
+              inputType: TextInputType.number,
+            ),
+            const SizedBox(
+              height: 20,
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                Button(
+                  text: 'Confirms',
+                  onPressed: () {
+                    submitAmountData();
+                  },
+                  padding: const EdgeInsets.symmetric(
+                    vertical: 16,
+                    horizontal: 38,
+                  ),
+                ),
+                TextButton(
+                  onPressed: () {
+                    Navigator.pop(context);
+                  },
+                  child: Text(
+                    'Cancel',
+                    style: Theme.of(context).textTheme.labelLarge!.copyWith(
+                          fontWeight: FontWeight.w600,
+                          color: const Color.fromARGB(255, 0, 140, 255),
+                        ),
+                  ),
+                )
+              ],
+            ),
+            const SizedBox(
```

Comment:
is this SizedBox necessary?
its the last content already

Reply:
SizedBox is used for having extra space in the bottom sir